src/recensio/plone/browser/review.py

```diff
@@ -45,18 +50,50 @@ def metadata_fields(self):
 
     @property
     def fields(self):
+        """Get all the fields of all schemata, including behaviors."""
         if self._fields is None:
-            self._fields = {}
+            self._fields = []
             schemata = iterSchemata(self.context)
             for schema in schemata:
                 for attr in schema.names():
-                    self._fields[attr] = schema.get(attr)
+                    self._fields.append(schema.get(attr))
         return self._fields
 
+    _widgets = None
+
+    @property
+    def widgets(self):
+        """Get all the widgets for all the fields fields in display mode.
+
+        Difference to the plone.dexterity DefaultView:
+        - Shows all widgets, regardless of their omit settings.
+        - Does not group the widgets in main and fieldset groups.
+
+        See also:
+        - plone.dexterity.browser.view.DefaultView
+        - plone.autoform.view.WidgetsView
+        - z3c.form.form.DisplayForm
+        """
+        if self._widgets is None:
+            form = DisplayForm(self.context, self.request)
```

Comment:
What if you inherit from `DisplayForm` instead of BrowserView?

Reply:
That was my first attempt. But I‌ withdraw that because:
- I had to flatten-out self.widgets as the individual widgets are grouped in fieldsets
- The plone.dexterity.browser.view.DefaultView did pollute the class with it's own properties, modifies `__call__` and so on...
- The review view broke, because it was tried to get the labels for some widgets which were not included due to the fields OMIT settings.